src/test/three-components/Renderer-spec.ts

```diff
@@ -103,15 +103,15 @@ suite('Renderer', () => {
       expect(!scene.isDirty).to.be.ok;
     });
 
-    test('does not render scenes marked as !isVisible', async function() {
-      scene.isVisible = false;
+    test('does not render scenes marked as not visible', async function() {
```

Comment:
This is really just a test of three.js at this point, but I guess it isn't hurting anything.